test/base/MockObligationsConnector.scala

```diff
@@ -18,14 +18,14 @@ package base
 
 import connectors.ObligationsConnector
 import models.obligations.PPTObligations
+import models.returns.TaxReturnObligation
 import org.mockito.ArgumentMatchers.any
-import org.mockito.Mockito.{when, reset}
+import org.mockito.Mockito.{reset, when}
```

Comment:
you may want to use the org.mockito.MockitoSugar for the when and especially the reset as the Mockito version giver a warning with scala 2.13.x

Reply:
this hasnt changed, its just optimised imports